docs/release-notes/8.16.asciidoc

```diff
@@ -0,0 +1,116 @@
+* Ensures that on Windows, {elastic-defend} uses https://www.elastic.co/security-labs/finding-truth-in-the-shadows[Intel CET and AMD Shadow Stacks] to collect call stacks, where supported. This improves performance and enables the detection of certain defense evasions. You can turn this feature off in {elastic-defend} <<adv-policy-settings,advanced policy settings>> ({kibana-pull}190553[#190553]).
+* Restore {elastic-defend}'s support for Windows Server 2012, which was removed in 8.13.0.
+* Improves {elastic-defend}'s caching to reduce memory usage on Windows.
+* Enhances {elastic-defend} by reducing the size of process events, which reduces excessive process ancestry entries and shortens the entity ID.
+* Improves the reliability and system resource usage of {elastic-defend}'s Windows network driver.
+
+[discrete]
+[[bug-fixes-8.16.0]]
+==== Bug fixes
+
+* Prevents an empty warning message from appearing for rule executions ({kibana-pull}186096[#186096]). 
+* Fixes an error that could occur during rule execution when the source index had a non-ECS-compliant text field ({kibana-pull}187673[#187673]). 
+* Removes unnecessary empty space below the title of the Open Timeline modal ({kibana-pull}188837[#188837]). 
+* Improves the Alerts table's performance ({kibana-pull}192827[#192827]). 
+* Removes the requirement that you have unnecessary {kib} {fleet} privileges to access some cloud security posture Findings ({kibana-pull}194069[#194069]).
+* Fixes an issue that could cause fields for all indices to appear when you tried to add a rule filter ({kibana-pull}194678[#194678]).
+* Fixes an {elastic-defend} bug where network event deduplication logic could incorrectly drop Linux network events.
+* Fixes an {elastic-defend} bug where Windows API events might be dropped if they contain Unicode characters that can't be converted to ANSI.
+* Ensures that {elastic-defend} does not emit an empty `memory_region` if it can't enrich a memory region in an API event. With this fix, {elastic-defend} removes these fields.
+* Fixes an {elastic-defend} bug where {elastic-defend} doesn't properly enrich Windows API events for short-lived processes on older operating systems that don't natively include this telemetry, such as Windows Server 2019. This might result in dropped or unattributed API events.
+* Fixes a bug that prevented host name uniformity with {beats} products. If you request for {elastic-defend} use the fully qualified domain name (FQDN) in the `host.name` field, {elastic-defend} now reports the FQDN exactly as the OS reports it, instead of lowercasing by default.
+* Fixes an {elastic-defend} bug in behavior protection alerts, where prevention alerts could mistakenly be labeled as detection alerts.
+* Fixes a bug that caused {elastic-defend} to crash if a Kafka connection is busy.
+* Fixes cases where Automatic Import could generate invalid processors containing array access ({kibana-pull}196207[#196207]).
```

Comment:
Can we please add 2 more.

- Improves Timeline Table performance when `Row Renderers` are switched on ( https://github.com/elastic/kibana/pull/193316).
- Fixes mis-alignment of Filter Control labels on Alert Page (https://github.com/elastic/kibana/pull/193316).

Reply:
Added in [0a3b585](https://github.com/elastic/security-docs/pull/5977/commits/0a3b585c1f2203bdfd77ccb31859b61b7d96b809) - thank you!